src/TBuffer.cpp

```diff
@@ -2417,7 +2439,7 @@ void TBuffer::translateToPlainText(std::string& incoming, const bool isFromServe
                         break;
                     case 2:
                         isTwoTCharsNeeded = true;
-                    // Fall-through
+                        [[clang::fallthrough]];
```

Comment:
How about we use this instead: https://infektor.net/posts/2017-01-19-using-cpp17-attributes-today.html#using-the-fallthrough-attribute

C++17 introduces the `[[fallthrough]]`.